components/content/LoadingImage/LoadingImage.css

```diff
@@ -0,0 +1,35 @@
+@import url('../../../styles/tokens/pulse.css');
+
+diamond-loading-img {
+  img {
+    visibility: hidden;
+  }
+  animation: pulse var(--diamond-pulse-duration) infinite linear;
+  background: linear-gradient(
+    90deg,
+    var(--diamond-color-grey-100) 25%,
+    var(--diamond-color-grey-300) 50%,
+    var(--diamond-color-grey-100) 75%
+  );
+  background-size: 200% 100%;
+  display: inline-block;
+
+  &[block],
+  &[block] img {
+    display: block;
+  }
+
+  &[responsive] {
```

Comment:
Is there a way of sharing this image code between components so it's not duplicated?

Reply:
Other than 'diamond-image-common' stylesheet that goes against BEM I couldn't see a reasonable way of doing this